client/lxd_images.go

```diff
@@ -33,6 +33,27 @@ func (r *ProtocolLXD) GetImages() ([]api.Image, error) {
 	return images, nil
 }
 
+// GetImagesAllProjects returns a list of images across all projects as Image structs.
+func (r *ProtocolLXD) GetImagesAllProjects() ([]api.Image, error) {
+	images := []api.Image{}
+
+	v := url.Values{}
+	v.Set("recursion", "1")
+	v.Set("all-projects", "true")
+
+	err := r.CheckExtension("images_all_projects")
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = r.queryStruct("GET", fmt.Sprintf("/images?%s", v.Encode()), nil, "", &images)
```

Comment:
Can we use `api.NewURL` here please. It has utils for setting query parameters.

Reply:
I don't see `api.NewURL` being used in the client. Do you mean `http.NewRequest`?